pkg/cloud/services/route53/route53.go

```diff
@@ -102,19 +104,38 @@ func (s *Service) associateResolverRules() error {
 		}
 		s.scope.Info("Got resolver rule associations", "associations", associations)
 
+		vpcId := s.scope.VPC()
+		describeVpcsInput := &ec2.DescribeVpcsInput{
+			VpcIds: []*string{&vpcId},
+		}
+		describeVpcsOutput, err := s.Ec2Client.DescribeVpcs(describeVpcsInput)
+		if err != nil {
+			return errors.Wrap(err, "failed to describe vpc")
+		}
+		vpcCidr := *describeVpcsOutput.Vpcs[0].CidrBlock
```

Comment:
the VPC range should be available in the AWSCluster no need to fetch it from AWS api